LICENSE.txt

```diff
@@ -0,0 +1 @@
+see COPYING
```

Comment:
if pypi really expects a LICENSE file, text of COPYING* should go to LICENSE then